tests/integration/suite/actors/reminders/rebalancing.go

```diff
@@ -241,15 +229,15 @@ func (i *rebalancing) Run(t *testing.T, ctx context.Context) {
 	}
 }
 
-type httpServer struct {
+type rebalancingHTTPServer struct {
```

Comment:
We don't need this change because `rebalancing` is inferred through the test (package) name.

Reply:
There is another HTTP server in the package.

Reply:
This will not be the case with `serialization` being made a subpackage.